### PR TITLE
Remove duplicated words

### PR DESCRIPTION
**Pull request by RuffaloLavoisier**
`be be` -> `be`

### CODE REVIEW

File: documentation/asciidoc/accessories/camera/libcamera_apps_post_processing_writing.adoc [hunk 1]
```diff
@@ -6,7 +6,7 @@ We are keen to accept and distribute interesting post-processing stages contribu
 
 ==== Basic Post-Processing Stages
 
-Post-processing stages have a simple API, and users can create their own by deriving from the `PostProcessingStage` class. The member functions that must be implemented are listed below, though note that some be be unnecessary for simple stages.
+Post-processing stages have a simple API, and users can create their own by deriving from the `PostProcessingStage` class. The member functions that must be implemented are listed below, though note that some be unnecessary for simple stages.
```
Comment (lurch) [inline documentation/asciidoc/accessories/camera/libcamera_apps_post_processing_writing.adoc:9]: This was probably _supposed_ to be "may be" ?

Comment (lurch) [reply documentation/asciidoc/accessories/camera/libcamera_apps_post_processing_writing.adoc:9]: (other two changes look fine)

Comment (RuffaloLavoisier) [reply documentation/asciidoc/accessories/camera/libcamera_apps_post_processing_writing.adoc]: @lurch 
Thanks.

File: documentation/asciidoc/accessories/camera/libcamera_apps_post_processing_writing.adoc [hunk 2]
```diff
@@ -6,7 +6,7 @@ We are keen to accept and distribute interesting post-processing stages contribu
 
 ==== Basic Post-Processing Stages
 
-Post-processing stages have a simple API, and users can create their own by deriving from the `PostProcessingStage` class. The member functions that must be implemented are listed below, though note that some be be unnecessary for simple stages.
+Post-processing stages have a simple API, and users can create their own by deriving from the `PostProcessingStage` class. The member functions that must be implemented are listed below, though note that may be unnecessary for simple stages.
```
Comment (lurch) [inline documentation/asciidoc/accessories/camera/libcamera_apps_post_processing_writing.adoc:9]: Try again. I believe it was supposed to be "though note that some may be unnecessary for simple stages."

Comment (RuffaloLavoisier) [reply documentation/asciidoc/accessories/camera/libcamera_apps_post_processing_writing.adoc]: @lurch 
Ah...I misunderstood. fixed it. :laughing:

**Review by lurch — APPROVED** [review]

**Review by aallan — APPROVED** [review]

**Pull request merged**